andrepereiradasilva/joomla-cms
# JHtml script/stylesheet/image improvements (PoC)

PoC for https://github.com/andrepereiradasilva/joomla-cms/pull/24
#### Summary of Changes
- Adds JHtml scripts/styles with autoversion in isis
- Adds JHtml scripts/styles with autoversion in protostar
#### Testing Instructions
- Use latest satging
- Apply patch https://github.com/andrepereiradasilva/joomla-cms/pull/24
- Apply this patch
- Check if all js and css are loaded properly in isis and in protostar
